rllib/execution/buffers/multi_agent_replay_buffer.py

```diff
@@ -282,5 +283,29 @@ def set_state(self, state: Dict[str, Any]) -> None:
         for policy_id in buffer_states.keys():
             self.replay_buffers[policy_id].set_state(buffer_states[policy_id])
 
+    @DeveloperAPI
+    def apply(
```

Comment:
I'll copy this over to the new replay buffers and also update our usage of `update_priorities()` to fit `apply()` I think.

Reply:
thanks, yeah we'll definitely need this on all of the actors that will be supported by the `asynchronous_parallel_requests` function